3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -343,11 +361,23 @@ inline int ovx_hal_mul(const T *a, size_t astep, const T *b, size_t bstep, T *c,
 {
     try
     {
+        int rounding_policy = VX_ROUND_POLICY_TO_ZERO;
+        if ((float)scale != 0x1.010102p-8)
```

Comment:
Unfortunately not. Actually I've started with 0x1.010102p-8 constant from OpenVX1.1 spec but MSVC complains about p character.
May be it could be replaced with (0x1.010102 / 256.0)?